spec/namespaces/indices.yaml

```diff
@@ -938,6 +938,24 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/indices.delete@200'
+  /{index}/_tier/warm:
+    post:
+      operationId: indices.tier_warm.0
+      x-operation-group: indices.tier
```

Comment:
the `x-operation-group` will be used to generate the client method. In this case it will be `client.indices.tier()`. Usually the function is a verb if it's performing some action. Is `tier` appropriate for this context?

Reply:
makes sense, thanks for your input, let me update this.